src/lib/utils/price.ts

```
@@ -93,4 +94,17 @@ async function calculateTicketPrice(
   return qty * ticketPriceArs;
 }
 
-export { calculateTicketPrice, convertSatsToArs };
+async function calculateCurrencyToSats(
+  currency: string,
+  amount: number
+): Promise<number> {
+  const YADIO_API_URL = `https://api.yadio.io/rate/${currency}/BTC`;
+  const response = await fetch(YADIO_API_URL);
+  const data = await response.json();
+  const rate = (data?.rate).toFixed(0);
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)